Library/Homebrew/cask/lib/hbc/audit.rb

```diff
@@ -50,6 +51,18 @@ def summary_header
 
     private
 
+    def check_untrusted_pkg
+      odebug "Auditing pkg stanza: allow_untrusted"
+
+      return if @cask.sourcefile_path.nil?
+
+      tap = Tap.select { |t| t.cask_file?(@cask.sourcefile_path) }.first
+      return if tap.nil? || tap.user != "caskroom"
```

Comment:
Does `@cask.tap` work here instead of `Tap.select`?

Reply:
This is used by another check.

https://github.com/Homebrew/brew/blob/6034caf9fb8e2b7ac86f97f2cf6c3ced621c8920/Library/Homebrew/cask/lib/hbc/audit.rb#L95-L99

Reply:
`@cask.tap` works, should I change both?

Reply:
Yes, change both.